add page for seminar series

mblue9 opened the pull request.

mblue9 committed:
add page for seminar series

lshep commented:
should we link this someplace off the main page or also a link on the event page?

mblue9 committed:
Add seminar series to home page

mblue9 committed:
Add seminar series to events page

mblue9 committed:
Add seminar series Dec 2 event

mblue9 commented:
yes, good idea! I’ve added it to the home page under “Connect with the Community” (an alternative could be in the Learn section, but we’d have to drop an item from there). I’ve also added it to the Events page

mblue9 committed:
Add link to Dec 2 seminar Youtube recording
